pkg/taxonomy/compile/transform.go

```diff
@@ -17,7 +17,8 @@ const nameKey = "name"
 // transform applies transformations over an input document to make it structural.
 // It requires a base document and the document to transform (mutable).
 // The codegenTarget can be set to true to apply more transformations to make the document
-//  more suitable for code generation tools.
+//
```

Comment:
Why are there empty lines in a middle of a comment?

Reply:
`go fmt` changed his comment formatting, as result all the changes in comments. 
Fixed this new line separation.

Reply:
There should not be a separation in this or other cases, just indent the last line correctly - it has an extra space. In other cases, there is an extra tab. All comments need to start at the same indentation.

Reply:
No, fmt adds a new line before lint tags. 
Please check now.